aea/cli/search.py

```diff
@@ -71,7 +71,7 @@ def search(ctx: Context, registry):
             # agent_config = ctx.agent_loader.load(fp)
             registry_directory = ctx.agent_config.registry_path
         except Exception:
-            registry_directory = os.path.join(ctx.cwd, DEFAULT_REGISTRY_PATH)
+            registry_directory = os.path.join("..", DEFAULT_REGISTRY_PATH)
```

Comment:
The original functionality is correct.

Reply:
Is that? Case: loading agent config fails from agent's folder, it leads us to `./packages` path inside the agent's directory which is not existing. Exactly the case we had on Friday's presentation.